pkg/certs/constants.go

```diff
@@ -140,6 +140,10 @@ const (
 	DefaultAPIServerBindAddress = "0.0.0.0"
 )
 
+func CertMap() map[string]string {
```

Comment:
Any caller of this will be able to modify the `certMap`.  Maybe clone the map if you don't want that, or just export the map directly?

Reply:
Good point. I'll restructure so that most of the logic moves to `pkg/certs` so there's no need anymore to export this.